src/basho_bench_stats_writer_csv.erl

```diff
@@ -133,6 +172,59 @@ measurement_csv_file({Label, _Op}) ->
     ok = file:write(F, <<"elapsed, window, n, min, mean, median, 95th, 99th, 99_9th, max, errors\n">>),
     F.
 
+dump_run_statistics(_RunStatsType=json) ->
+    %% Assemble Json snippets, building one document containing an entry for
+    %% each OP, then write it out to runStatistics.json
+    RunMetrics = erlang:get(run_metrics),
+
+    %% Open metrics file
+    TestDir = basho_bench:get_test_dir(),
+    {ok, GlobalMetricsFile} = file:open(
+        filename:join([TestDir, "runStatistics.json"]),
```

Comment:
The rest of the result files are using underscore separators rather than camel case, let's stick with that pattern.